demosplan/DemosPlanCoreBundle/Logic/Import/Statement/StatementFromRowBuilderWithZipSupport.php

```diff
@@ -209,11 +260,33 @@ public function setFeedback(Cell $cell): ?ConstraintViolationListInterface
 
     public function setNumberOfAnonymVotes(Cell $cell): ?ConstraintViolationListInterface
     {
-       return $this->baseStatementFromRowBuilder->setNumberOfAnonymVotes($cell);
+        return $this->baseStatementFromRowBuilder->setNumberOfAnonymVotes($cell);
+    }
+
+    public function setFileReferences(?Cell $cell): ?ConstraintViolationListInterface
+    {
+        $this->fileReferences = $cell;
+
+        return null;
     }
 
     public function buildStatementAndReset(): StatementInterface|ConstraintViolationListInterface
     {
+        $violations1 = $this->handleFileReferences();
+        if (null !== $violations1 && 0 !== $violations1->count()) {
+            return $violations1;
+        }
+
+        $violations2 = $this->handleOriginalFileReferences();
+        if (null !== $violations2 && 0 !== $violations2->count()) {
+            return $violations2;
+        }
```

Comment:
We want to show as many violations on the error page as possible. If there is no interdependence between these two methods, I suggest to execute them both and return all resulting violations.